packages/@lwc/babel-plugin-component/src/constants.js

```diff
@@ -42,6 +42,7 @@ const LWC_API_WHITELIST = new Set([
     'isComponentConstructor',
     'readonly',
     'register',
+    'setFeatureFlag',
```

Comment:
This would open up toggle access to components as well as services. I need to verify this, but if services can access `setFeatureFlag` without whitelisting it here, then we could remove this line and instead whitelist a test version (e.g., `setFeatureFlagForTesting`) that throws when invoked in production instead. This way, services can invoke `setFeatureFlag` and `setFeatureFlagForTesting` can be used in tests.